docs/operators_and_observables.ipynb

```diff
@@ -779,6 +779,29 @@
     "The pattern for computing $\\text{Tr} [ \\rho O ]$ is shown below."
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "7d55b4675db8"
+   },
+   "outputs": [],
+   "source": [
+    "!pip install nest-asyncio\n",
+    "# for running collector under jupyter environment\n",
+    "import nest_asyncio\n",
+    "nest_asyncio.apply()   "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "5d8e48230d73"
+   },
+   "source": [
+    "**Warning: If you run this notebook in colab environment with \"Run all\", the process may hang forever or you may encounter crash. Please manually run this cell either press the play button to the left of the code, or use the keyboard shortcut \"Command/Ctrl+Enter\".**"
```

Comment:
A few nits:
- `in colab environment` -> `in a colab environment`
- `either press...or use` -> `either by pressing...or using`
- Instead of `the process`, I recommend saying `the cell below` to be abundantly clear which cell is the source of the issue. (As it is, it's unclear if this refers to the cell above or below)

Reply:
Done.